src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .reduce((acc, curr) => {
+      const [key, value] = curr.split(':').map((str) => str.trim());
+
+      acc[`${key}`] = value;
```

Comment:
Using template literals for object keys is unnecessary here. You can simply use `acc[key] = value;` instead of `acc[`${key}`] = value;`.

Reply:
> Using template literals for object keys is unnecessary here. You can simply use `acc[key] = value;` instead of `acc[`${key}`] = value;`.

Good point